src/main/java/ch/wisv/areafiftylan/service/SeatServiceImpl.java

```diff
@@ -47,10 +47,10 @@ public SeatmapResponse getAllSeats() {
 
     @Override
     public SeatmapResponse getSeatGroupByName(String groupname) {
-        List<Seat> seatGroup = seatRepository.findBySeatGroup(groupname);
+        List<Seat> seatGroup = seatRepository.findBySeatGroupIgnoreCase(groupname);
 
         Map<String, List<Seat>> seatMapResponse = new HashMap<>();
-        seatMapResponse.put(groupname, seatGroup);
+        seatMapResponse.put(seatGroup.get(0).getSeatGroup(), seatGroup);
```

Comment:
Border-case seatGroup is empty throws an ArrayOutOfBounds error here. Even though I'm sure this is not likely to happen I think it's better to make sure no undefined errors can be thrown.